src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -71,20 +71,27 @@
         {% set label = name|humanize %}
       {%- endif -%}
     {%- endif -%}
-    <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>{{ translation_domain is same as(false) ? label|raw : label|raw }}
-    {% if label_attr['tooltip'] is defined %}
-      {% set placement = label_attr['tooltip_placement'] is defined ? label_attr['tooltip_placement'] : 'top' %}
-      <i class="icon-question" data-toggle="pstooltip" data-placement="{{ placement }}"
-         title="{{ label_attr['tooltip'] }}"></i>
-    {% endif %}
+    {# options 'comes' from service: prestashop.admin.credit_slip_options.form_handler in .yml file and is added as additional field; #}
```

Comment:
So we need to make "buggy" modification in the template because the service was weirdly configured 😅

This feels weird.. If there is a problem with the `options` field of `prestashop.admin.credit_slip_options.form_handler` then I guess there will be a similar issue with the `configuration` field of `prestashop.admin.mail_theme.form_handler` or the `shop_logos` of `prestashop.admin.shop_logos_settings.form_handler`, and so on..

We can't keep piling ignore cases as we migrate the configuration forms..